financial-connections/src/main/java/com/stripe/android/financialconnections/ui/FinancialConnectionsSheetNativeActivity.kt

```diff
@@ -146,43 +144,37 @@ internal class FinancialConnectionsSheetNativeActivity : AppCompatActivity(), Ma
                 viewModel.onBackClick(navController.currentDestination?.pane)
                 if (navController.popBackStack().not()) viewModel.onBackPressed()
             }
-            FinancialConnectionsModalBottomSheetLayout(
-                sheetState = bottomState,
-                sheetContent = {
-                    exitModal?.let {
-                        ExitModal(
-                            loading = it.loading,
-                            description = it.description,
-                            onCancel = { coroutineScope.launch { bottomState.hide() } },
-                            onExit = viewModel::onCloseConfirm
-                        )
-                    }
-                },
-                content = {
-                    NavHost(
-                        navController,
-                        startDestination = initialDestination.fullRoute,
-                    ) {
-                        composable(Destination.Consent)
-                        composable(Destination.ManualEntry)
-                        composable(Destination.PartnerAuth)
-                        composable(Destination.InstitutionPicker)
-                        composable(Destination.AccountPicker)
-                        composable(Destination.Success)
-                        composable(Destination.Reset)
-                        composable(Destination.Error)
-                        composable(Destination.AttachLinkedPaymentAccount)
-                        composable(Destination.NetworkingLinkSignup)
-                        composable(Destination.NetworkingLinkLoginWarmup)
-                        composable(Destination.NetworkingLinkVerification)
-                        composable(Destination.NetworkingSaveToLinkVerification)
-                        composable(Destination.LinkAccountPicker)
-                        composable(Destination.BankAuthRepair)
-                        composable(Destination.LinkStepUpVerification)
-                        composable(Destination.ManualEntrySuccess)
-                    }
+            ModalBottomSheetLayout(
+                bottomSheetNavigator = bottomSheetNavigator,
+                sheetBackgroundColor = FinancialConnectionsTheme.v3Colors.backgroundSurface,
+                sheetShape = RoundedCornerShape(8.dp),
```

Comment:
Nit: Do we want to round _all_ corners or just the top ones? I’d argue that the latter makes more sense, but happy to chat.

Reply:
forgot to use our connections modal layout also for navigation -> https://github.com/stripe/stripe-android/pull/7882/commits/c225f9b87d79a1a25e6e8fafd9e00b8abfc3c792